docs/provisioning-methods-and-credentialszip.adoc

```diff
@@ -0,0 +1,63 @@
+// Copy of this topic: https://docs.ota.here.com/concepts/provisioning-methods-and-credentialszip.html adapted for aktualizr repo. Replaces duplicate "credentials.adoc".
+
+= Provisioning methods and credentials.zip
+
+If you're trying to work on integrating {product-name} into your device build, it may be helpful to have a little bit of reference information on how the different provisioning methods work together, and what exactly the magic sauce inside `credentials.zip` is.
+
+== `credentials.zip` file format
+
+First, a table:
+
+The following files are present in credentials.zip:
```

Comment:
Weird to have two introductory lines here. I think one should be cut.

Reply:
Fixed